src/WikibaseRdfExtension.php

```diff
@@ -106,4 +106,11 @@ private function newSaveMappingsApi(): SaveMappingsApi {
 		);
 	}
 
+	/**
+	 * @return string[]
+	 */
+	private static function getAllowedPredicates(): array {
+		return (array)MediaWikiServices::getInstance()->getMainConfig()->get( 'WikibaseRdfPredicates' );
```

Comment:
Is there a preferred way of coercing the MW config retrieval into the correct types (like a string array here) or do we simply silence the CS warnings?

Reply:
I do not know of a way to do that, or of one to add such support to this type of interface. So just silence the warnings. They are basically telling us the MW interface is not type safe, which is something we already know. I think it is reasonable to just disable CS for WikibaseRdfExtension.php altogether. Same for `EntryPoints/`